src/validate_date_format.py

```diff
@@ -1,28 +1,52 @@
 from datetime import datetime
 import sys
 
-def validate_dates(str_date):
+# Validate ISO 8601 Format
+def validate_dates_iso8601format(str_date):
     try:
         datetime.strptime(str_date,'%Y-%m-%d')
         return str_date
     except ValueError:
         return False
 
-def validate_datetime(str_date):
+def validate_datetime_iso8601format(str_date):
```

Comment:
It's better if we update the function name to `iso_8601_datetime` or `iso_8601_date`